docs/src/howto-guides/008_config_local_mqtt_port.md

```diff
@@ -32,7 +32,8 @@ This prints out the port that has been set.
 
 Using `tedge` command connect to the desired cloud as below.
 
-This will force all the services (mosquitto, tedge_mappers) to use newely set port.
+This will force all the services (mosquitto, tedge-mapper-c8y.service,tedge-mapper-az.service sm_agent,
```

Comment:
Just very minor grammar.

```suggestion
This will force all the services (mosquitto, tedge-mapper-c8y.service, tedge-mapper-az.service, sm_agent,
```

Reply:
fixed